networkit/cpp/io/FastKONECTGraphReader.cpp

```diff
@@ -0,0 +1,270 @@
+/*
+ * FastKONECTGraphReader.cpp
+ *
+ *  Created on: 11.05.2018
+ *      Author: Roman Bange
+ *
+ * The KONECT format is described in detail in
+ * http://konect.uni-koblenz.de/downloads/konect-handbook.pdf
+ *
+ */
+
+#include <fcntl.h>
+#include <sys/stat.h>
+#include <sys/mman.h>
+#include <unistd.h>
+
+#include "../auxiliary/Log.h"
+
+#include "FastKONECTGraphReader.h"
+
+namespace NetworKit{
+	FastKONECTGraphReader::FastKONECTGraphReader(const bool& remapNodes, MultipleEdgesHandling handlingmethod):
```

Comment:
Same as in #168 - why is the `bool` passed by reference?

Reply:
Fixed in https://github.com/kit-parco/networkit/pull/169/commits/2af7dbc977feb8f954a6706ba93c009295ecf4e0